numpy/core/src/umath/ufunc_object.c

```diff
@@ -2863,37 +2863,47 @@ PyUFunc_Reduce(PyUFuncObject *ufunc, PyArrayObject *arr, PyArrayObject *out,
         return NULL;
     }
 
-    /* Get the identity */
+    /* Check if identity makes sense */
```

Comment:
Not really accurate. `_get_identity` only fails if something really bad has happened, either:

* Someone has created an invalid ufunc
* `PyInt_FromLong(1)` failed

I'd prefer to leave the original comment